using_listwheelscrollview/lib/roundcontainer.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class NewWidget extends StatelessWidget {
+  final Widget l;
+  final String s;
+  double radius = 8;
+  NewWidget({this.size, this.l, this.s});
+  final double size;
+
+  @override
+  Widget build(BuildContext context) {

```

Comment:
Split build function into sub functions so it is easier to consume. These example apps are for beginners. Putting everything in the same nested function is hard to read.

Reply:
Also add code comments to explain what each function is doing, or name the function correctly and descriptive.

Reply:
ok sir.

Reply:
sir , this is my first pr. so I apologize😅 for any mistake in the way things are done.